src/components/Drinks.jsx

```diff
@@ -1,41 +1,33 @@
 import React from 'react';
-import styled from 'styled-components';
-import { Card } from 'semantic-ui-react';
+import Card from '../components/atoms/Card';
 import { graphql, StaticQuery } from 'gatsby';
 
 class Drinks extends React.Component {
   renderDescription = drinks => {
     return (
-      <Card.Description>
+      <>
         Elke week is er een gratis borrel! Deze periode is de borreldag: <br />
         <b>
           Elke {drinks.day} vanaf {drinks.time} in de {drinks.location}!
         </b>
-      </Card.Description>
+      </>
     );
   };
 
   render() {
     return (
-      <DrinksWrapper>
+      <>
         <div className="drinksCard">
```

Comment:
I think you can remove the `className` attribute, if you're cleaning up anyway

Reply:
Good one

Reply:
Fix incoming